packages/flame/lib/src/cache/cache.dart

```diff
@@ -0,0 +1,2 @@
+export 'memory_cache.dart';
+export 'value_cache.dart';
```

Comment:
Is this file needed? Can't they be exported directly in the other file?

Reply:
They could, but I think it is good to have this barrel file, so we can use it internally.

I don't mind removing it if you feel too strong about it though.

Reply:
I don't know how it usually is done, but can't we just use the other exported file internally?

Reply:
Yeah, but having a barrel we can import everything related to cache on a single import, otherwise we will need to individually import each file

Reply:
If the exports moved to `packages/flame/lib/cache.dart` and this file removed, wouldn't it have the same effect?

Reply:
Or shouldn't the externally exported files be used internally? There is something I'm missing I think. :sweat_smile:

Reply:
> If the exports moved to `packages/flame/lib/cache.dart` and this file removed, wouldn't it have the same effect?

Yes! And I would honestly prefer that. But we have been doing relatively imports internationally, so that is why I created this internal barrel